examples/keycloak/README.md

```diff
@@ -0,0 +1,113 @@
+# Keycloak Setup Guide
+
+In order to use the Tenzir Platform with a Keycloak instance
+as Identity Provider, it is necessary to do some additional
+manual configuration.
+
+This document is intended to guide you through this process.
+
+We assume that you already have a Keycloak instance up
+and running that is reachable with a browser, for example
+by running the `keycloak` service from the docker compose file
+in this example. The same configuration steps apply when using
+an external instance.
+
+## Setting up the Keycloak Instance
+
+### Admin Login
+
+Navigate to the keycloak instance and log in as a user with admin permissions.
+If you use the bundled keycloak instance, the initial username is `admin`
+and password is `changeme`.
+
+Remember to change the default password after logging in for the first time,
+and to set up 2-factor authentication for your admin user.
+
+### Create a new `tenzir` realm (optional)
+
+This step is not required. It can make sense if you already have
+an existing keycloak realm configured that you want to keep separate,
+or if you want to change the name of the default `master` realm.
+
+If you do, you'll need to update the `TENZIR_PLATFORM_OIDC_ISSUER_URL`
+in the docker compose config to point to the new realm instead.
```

Comment:
This just says that I can do it, but it doesn't tell me why I would do it.

Reply:
Did you notice the sentence before?

> It can make sense if you already have an existing keycloak realm configured that you want to keep separate, or if you want to change the name of the default `master` realm.

Reply:
Slight rewording suggestion to make it more actionable:

> Keycloak defaults to the `master` realm. If you want to use a different `realm`, or already have an existing one, update the `TENZIR_PLATFORM_OIDC_ISSUER_URL` to point to the new real instead.